e2e/playwright/pay.spec.ts

```diff
@@ -0,0 +1,359 @@
+import { test, expect, type Page } from "@playwright/test";
+import { readFileSync } from "fs";
+import path from "path";
+import { gqlAdmin, insertTeam, insertTestUser } from "./utils";
+
+import team from "./assets/buckinhamshire.json";
+import ldcflow from "./assets/ldcFlow.json";
+import publishedFlows from "./assets/publishedFlowLDC.json";
+import operation from "./assets/operationLDC.json";
```

Comment:
If this test is intended to test the Gov Pay integration, I'm not sure I understand why it's important for flow content to reflect a full LDC flow? Why not just a simple published flow that has a SetValue (in order to set amount = 1), Pay (reads from "amount" passport field), then Confirmation (because confirmation page should display pay reference number on success) and **focus on testing different Pay _states_** (eg success, failure & retry, etc)? 

This seems to introduce a fair bit of noise to replicate production content (if we do want to use a LDC flow, can we point to production or staging rather than storing static JSON here?) and as a side effect has to deal with complex non-Pay functionalities like `uploadFileToInput`, `navigateToPayComponent`, etc which I imagine make it more difficult to determine when this test fails for a Pay-specific reason?

Reply:
Also more generally, I'd be interested to see what an alternative that uses assets downloaded from the DB looks like. This would allow us to test journeys still worked if the structure or data changed.

1. Flows / teams are generated on prod
2. Data copied onto test environment's db
3. Download them as assets (or directly query...?)
4. Run tests against "real" data

I don't think this needs to be achieved in this PR but I would be curious if this is a model we could use. For example, if we changed for the `teams` table was structured we wouldn't need to manually update our assets as part of that change.